working_dir.go

```diff
@@ -181,13 +190,16 @@ func (wd *WorkingDir) RequireCreatePlan(t TestControl) {
 // this will apply the saved plan. Otherwise, it will implicitly create a new
 // plan and apply it.
 func (wd *WorkingDir) Apply() error {
-	args := []string{"apply"}
+	args := []string{"apply", "-refresh=false"}
```

Comment:
I'm not a `refresh` expert but would this be a bad idea in that we aren't testing with terraform's default behavior? How did our existing testing driver behave? In general I will approve I just want to make sure we aren't creating a footgun